helmfile/ibft2/charts/besu-monitoring/templates/prometheus-service.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "besu-monitoring.fullname" . }}-prometheus
+  labels:
+    app: {{ template "besu-monitoring.name" . }}
+    chart: {{ template "besu-monitoring.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    namespace: monitoring
+    component: prometheus
+  namespace: monitoring
+spec:
+  type: NodePort
+  selector:
+    component: prometheus
+  ports:
+    - port: 9090
+      targetPort: 9090
+      nodePort: 30090
```

Comment:
Should this be configurable via values?

Reply:
Yep, we can do this one is a seperate PR for the lot